server.js

```diff
@@ -0,0 +1,96 @@
+const net = require('net');
+const fs = require('fs');
+
+let theDate = new Date().toUTCString();
+let contentType = ' ';
+let contentLength = 0;
+let body = '';
+
+function response(contentType, contentLength, body, theDate) {
+  return `HTTP/1.1 200 OK
+          Server: jatinku/1.0.0 
+          Date: ${theDate}
+          Content-Type: ${contentType}
+          Content-Length: ${contentLength}
+
+          ${body}
+          `;
+}
+
+const server = net.createServer(socket => {
+    socket.setEncoding('utf8');
+    socket.on('data', data => {
+
+      let dataIndex = data.slice(0, data.indexOf('H') - 1);
+
+      if (dataIndex === 'GET /index.html') {
```

Comment:
Although this work, you should parse out the `GET` from this so that you're just left with the path. Then you can just put the path into the readFile below and just have one case that can handle any file rather than making a separate case for each one.